src/main/java/uk/gov/pay/card/resources/HealthCheckResource.java

```diff
@@ -33,24 +35,20 @@ public HealthCheckResource(Environment environment) {
     public Response healthCheck() {
         SortedMap<String, HealthCheck.Result> results = environment.healthChecks().runHealthChecks();
 
-        Map<String, Map<String, Boolean>> response = getResponse(results);
-
-        boolean healthy = results.size() == results.values()
+        Map<String, Map<String, Boolean>> response = results.entrySet()
                 .stream()
-                .filter(HealthCheck.Result::isHealthy)
-                .count();
+                .collect(Collectors.toMap(Map.Entry::getKey,
+                        healthchechResult -> ImmutableMap.of(
```

Comment:
Your spelling of “healthcheck” isn’t healthy!

Also, looks like we could replace `ImmutableMap.of(…)` with `Collections.singletonMap(…)`.